django_hatstall/identities/templates/organizations/organizations.html

```diff
@@ -56,25 +56,6 @@ <h2>Add organization</h2>
         </form>
     </div>
 
-    <div class="col-sm-4">
-        <h1>Domains and organizations</h1>
-        <table class="table table-striped" id="domsTable">
-            <thead class="thead-dark">
-                <tr>
-                    <th>Name</th>
-                    <th>Domain</th>
-                </tr>
-            </thead>
-            <tbody>
-                {%  for domain in domains %}
-                <tr>
-                    <td>{{domain.organization.name}}</td>
-                    <td>{{domain.domain}}</td>
-                </tr>
-                {% endfor %}
-            </tbody>
-        </table>
-    </div>
```

Comment:
For next steps, it will be better to have this kind of features in specific templates, so we don't loose the feature, just don't include it in the template.